CONTRIBUTING.md

```diff
@@ -22,3 +32,9 @@ When you are ready to start work on a issue:
 3. Develop the feature and push it to your local repo.
 
 4. Create Pull Request(PR). Make sure to comment the issue that your PR is supposed to solve.
+
+## Community
+
+Discussions about the CircuitVerse take place on the repository's [Issues](https://github.com/CircuitVerse/CircuitVerse/issues) and [Pull Requests](https://github.com/CircuitVerse/CircuitVerse/pulls) sections. Anybody is welcome to join these conversations. There is also a [chat room](https://gitter.im/CircuitVerse/community) for regular updates.
+
+Wherever possible, do not take these conversations to private channels. Keeping communication public means everybody can benefit and learn from the conversation.
```

Comment:
>We encourage people to talk on public platforms for the maximum participation and try to avoid private messages/conversations. 

Something along these lines? Make it more readable. Thoughts?

Reply:
Should i bold them?

Reply:
Not required.

Reply:
so should i add these lines?

> > We encourage people to talk on public platforms for the maximum participation and try to avoid private messages/conversations.